eng/scripts/Verify-NeedToRelease.ps1

```diff
@@ -0,0 +1,42 @@
+param (
+  $PackageName,
+  $ServiceDirectory,
+  $repoId
+)
+
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+
+$apiUrl = "https://api.github.com/repos/$repoId"
+Write-Host "Using API URL $apiUrl"
+
+# VERIFY CHANGELOG
+$PackageProp = Get-PkgProperties -PackageName $PackageName -ServiceDirectory $ServiceDirectory
+$changeLogEntries = Get-ChangeLogEntries -ChangeLogLocation $PackageProp.ChangeLogPath
+$changeLogEntry = $changeLogEntries[$PackageProp.Version]
+
+if (!$changeLogEntry)
```

Comment:
Why do we want to skip releasing if there is no changelog? Why wouldn't we just error?

Reply:
This check release step is just used to check whether next release stage need to trigger to avoid manual trigger step. Every release validation will be done in the next release stage.